### PR TITLE
fix: load-more button added in place of infinite scroll

### PR DESCRIPTION
**Pull request by mehaknasir**
Ticket ID : https://2u-internal.atlassian.net/browse/INF-275

- To resolve accessibility issues with loading more threads a visible element is added as load more button to help resolve the issue. 
Screenshot 2022-06-24 at 6 28 14 PM

- also added aria-level properties to post UI components

### CODE REVIEW

**Review by muhammadadeeltajamul — APPROVED** [review]

**Review by awais-ansari — CHANGES_REQUESTED** [review]
Please add `Show more` button in the learners area too.

File: src/discussions/posts/PostsView.jsx [hunk 1]
```diff
@@ -1,112 +1,16 @@
...
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 
-import { useDispatch, useSelector } from 'react-redux';
+import { useSelector } from 'react-redux';
 
-import { AppContext } from '@edx/frontend-platform/react';
-import { Spinner } from '@edx/paragon';
-
-import ScrollThreshold from '../../components/ScrollThreshold';
```
Comment (asadazam93) [inline src/discussions/posts/PostsView.jsx:9]: Should we remove ScrollThreshold component? Or is it used somewhere else?

**Review by asadazam93 — APPROVED** [review]
Just one question. Rest looks good

**Comment by asadazam93** [comment]
@mehaknasir Can you please link the related ticket?

**Comment by mehaknasir** [comment]
> @mehaknasir Can you please link the related ticket?

I think it was in use in some other places.  but I am making a change as suggested by Awais so will look if there are any more places where its being used.

**Review by awais-ansari — APPROVED** [review]

**Pull request merged**